From 790604ebcb97323af817ef909b555dc6d31ad5fb Mon Sep 17 00:00:00 2001
From: ShenZhengyi <szy@rock-chips.com>
Date: Mon, 6 Jul 2015 09:54:36 +0800
Subject: [PATCH] tve:gm7122: When gm7122 init, do not switch fb.

	it can avoid green screen, when HDMI switch to CVBS.

Signed-off-by: ShenZhengyi <szy@rock-chips.com>
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/video/rockchip/tve/gm7122/gm7122_tve.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 mode change 100644 => 100755 drivers/video/rockchip/tve/gm7122/gm7122_tve.c

diff --git a/drivers/video/rockchip/tve/gm7122/gm7122_tve.c b/drivers/video/rockchip/tve/gm7122/gm7122_tve.c
old mode 100644
new mode 100755
index 07815325ef49..47642c762d94
--- a/drivers/video/rockchip/tve/gm7122/gm7122_tve.c
+++ b/drivers/video/rockchip/tve/gm7122/gm7122_tve.c
@@ -433,8 +433,8 @@ static int gm7122_tve_probe(struct i2c_client *client,
 	if (cvbsformat >= 0) {
 		gm7122_tve->mode =
 			(struct fb_videomode *)&gm7122_cvbs_mode[cvbsformat];
-		gm7122_tve->enable = 1;
-		tve_switch_fb(gm7122_tve->mode, 1);
+		/*gm7122_tve->enable = 1;
+		tve_switch_fb(gm7122_tve->mode, 1);*/
 	} else {
 		gm7122_tve->mode = (struct fb_videomode *)&gm7122_cvbs_mode[1];
 	}
-- 
2.35.3

